src/components/calcite-input/calcite-input.scss

```diff
@@ -24,7 +24,7 @@
   & textarea {
     @apply text--2h
     py-1
-    px-2  
```

Comment:
@jcfranco I didn't run prettier or anything but some post-commit hook is doing this. It seems to have happened here: https://github.com/Esri/calcite-components/commit/f5270f8085b8a08955e65cdc9fd2a1a39c7e5e0b#diff-4185271e827d2b26e19a1a16201fb3dc4a31448b796c61107967bb5cfe2ca722R27

Maybe some are skipping these hooks???

Reply:
For posterity, the auto-linting NPM setup script was not using the recommended lifecycle script hook and was easy to miss. https://github.com/Esri/calcite-components/commit/566a991a96e7605bc1f8ff2c7d7f50614fbe528d updates it to set up on `npm install`.